cms/templates/index.html

```diff
@@ -473,68 +442,11 @@ <h4 class="title">${_('Your Course Creator Request Status:')}</h4>
-          <li class="course-item" data-course-key="${course_info['course_key']}">
-            <a class="course-link" href="${course_info['url']}">
-              <h3 class="course-title">${course_info['display_name']}</h3>
-
-              <div class="course-metadata">
-                <span class="course-org metadata-item">
-                  <span class="label">${_("Organization:")}</span> <span class="value">${course_info['org']}</span>
-                </span>
-                <span class="course-num metadata-item">
-                  <span class="label">${_("Course Number:")}</span>
-                  <span class="value">${course_info['number']}</span>
-                </span>
-                <span class="course-run metadata-item">
-                  <span class="label">${_("Course Run:")}</span> <span class="value">${course_info['run']}</span>
-                </span>
-              </div>
-            </a>
-
-            <ul  class="item-actions course-actions">
-              % if allow_course_reruns and rerun_creator_status and course_creator_status=='granted':
-              <li class="action action-rerun">
-                <a href="${course_info['rerun_link']}" class="button rerun-button">${_("Re-run Course")}</a>
-              </li>
-              % endif
-              <li class="action action-view">
-                <a href="${course_info['lms_link']}" rel="external" class="button view-button">${_("View Live")}</a>
-              </li>
-            </ul>
-          </li>
-          %endfor
-        </ul>
-      </div>
+      <div class="archived-courses react-archived-course-listing archived-courses-tab"></div>
       %endif
 
-      %if len(libraries) > 0:
-      <div class="libraries libraries-tab">
-        <ul class="list-courses">
-          %for library_info in sorted(libraries, key=lambda s: s['display_name'].lower() if s['display_name'] is not None else ''):
-          <li class="course-item">
-            <a class="library-link" href="${library_info['url']}">
-              <h3 class="course-title">${library_info['display_name']}</h3>
-
-              <div class="course-metadata">
-                <span class="course-org metadata-item">
-                  <span class="label">${_("Organization:")}</span> <span class="value">${library_info['org']}</span>
-                </span>
-                <span class="course-num metadata-item">
-                  <span class="label">${_("Course Number:")}</span>
-                  <span class="value">${library_info['number']}</span>
-                </span>
-                % if not library_info["can_edit"]:
-                  <span class="extra-metadata">${_("(Read-only)")}</span>
-                % endif
-              </div>
-            </a>
-          </li>
-          %endfor
-        </ul>
-      </div>
+      %if len(libraries) > 0 or optimization_enabled:
```

Comment:
The "do you want to create your first library" messaging shouldn't show if the org optimization is enabled (it was an oversight when the feature was extended to include libraries). Here is how it looks now when org is not specified:

![image](https://user-images.githubusercontent.com/484484/28935420-8987a39c-7852-11e7-9f1e-a0b36426a160.png)